test/parallel/test-debug-prompt.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const assert = require('assert');
+const common = require('../common');
+const spawn = require('child_process').spawn;
+
+const timeoutId = setTimeout(function() {
```

Comment:
Yea, if you don't need it, I'd remove it. They add complexity to the code and timers have been a source of annoyance on the CI.